systems-state-manager/nisystemsstate.yml

```diff
@@ -0,0 +1,999 @@
+        example: I've changed the state
+
+  StateResponse:
+    description: System state object.
+    title: State response
+    type: object
+    properties:
+      id:
+        type: string
+        description: The ID of the state.
+        example: 5bc4463af0f06f4585003ae6
+      name:
+        type: string
+        description: The name of the state.
+        example: crio9064State
+      fileName:
+        type: string
+        description: The file name of the state.
+        example: crio9064State
+      description:
+        type: string
+        description: The description of the state.
+        example: crio9064 Setup State
+      distribution:
+        $ref: '#/definitions/StateDistribution'
+      architecture:
+        $ref: '#/definitions/StateArchitecture'
+      createdTimestamp:
+        description: ISO-8601 formatted timestamp specifying the state creation date.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+      lastUpdatedTimestamp:
+        description: ISO-8601 formatted timestamp specifying the last date that the state has been updated.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+      feeds:
+        type: array
+        items:
+          $ref: '#/definitions/Feed'
+      packages:
+        type: array
+        items:
+          $ref: '#/definitions/Package'
+      containsExtraOperations:
+        type: boolean
+        description: Whether the state containts extra operations beside feeds and packages or not.
+        example: true
+    required:
+      - id
+      - name
+      - fileName
+      - distribution
+      - architecture
+      - createdTimestamp
+      - lastUpdatedTimestamp
+      - containsExtraOperations
+
+  StateVersion:
+    description: A given version of a state.
+    title: State Version
+    type: object
+    properties:
+      version:
+        type: string
+      description:
+        type: string
+        description: The description of this specific change.
+    required:
+      - version
+    example:
+      version: 26783106b56d87c5ef6c70c549b82e4ab004e590
+      description: added package
+
+  StateHistoryResponse:
+    description: The history of a state.
+    title: State History Response
+    type: object
+    properties:
+      totalCount:
+        type: integer
+        description: Total number of versions of a state.
+      versions:
+        type: array
+        items:
+          $ref: '#/definitions/StateVersion'
+    example:
+      totalCount: 2
+      versions:
+        - version: 4d514a6b31ee6ed72ae509c360be8dca6e68ffb0
+          description: initial version
+        - version: 0520222b9e7edae30c37c244dbce0deb23982c11
+          description: added new package
+    required:
+      - totalCount
+      - versions
+
+  Feed:
+    description: Object defining a feed containing the name, url and whether is enabled/compressed or not.
```

Comment:
"Object defining a feed, which contains the name, url, and Booleans for whether the feed is enabled and compressed."?